sklearn/linear_model/_logistic.py

```diff
@@ -993,7 +1121,7 @@ def _log_reg_scoring_path(X, y, train, test, pos_class=None, Cs=10,
                           dual=False, intercept_scaling=1.,
                           multi_class='auto', random_state=None,
                           max_squared_sum=None, sample_weight=None,
-                          l1_ratio=None):
+                          l1_ratio=None, precondition=True):
```

Comment:
docstring should be updated, possibly referencing the docstring of the estimator class for full details.

Reply:
So my proposal was to remove this parameter and always have it true. We already have too many parameters... But we can also keep it. I'm not sure it's ever beneficial, though.